lib/xeroizer/partner_application.rb

```diff
@@ -13,21 +13,17 @@ class PartnerApplication < GenericApplication
       # @param [String] consumer_key consumer key/token from application developer (found at http://api.xero.com for your application).
       # @param [String] consumer_secret consumer secret from application developer (found at http://api.xero.com for your application).
       # @param [String] path_to_private_key application's private key for message signing (uploaded to http://api.xero.com)
-      # @param [String] ssl_client_cert client-side SSL certificate file to use for requests
-      # @param [String] ssl_client_key client-side SSL private key to use for requests
       # @param [Hash] options other options to pass to the GenericApplication constructor
       # @return [PartnerApplication] instance of PrivateApplication
-      def initialize(consumer_key, consumer_secret, path_to_private_key, ssl_client_cert, ssl_client_key, options = {})
+      def initialize(consumer_key, consumer_secret, path_to_private_key, options = {})
         default_options = {
-          :xero_url         => 'https://api-partner.network.xero.com/api.xro/2.0',
-          :site             => 'https://api-partner.network.xero.com',
+          :xero_url         => 'https://api.xero.com/api.xro/2.0',
```

Comment:
Looks like they changed the urls and no longer need the certificate and key path (below). Here is the page explaining it: https://developer.xero.com/documentation/auth-and-limits/entrust-certificate-deprecation/